.github/workflows/run-renovate.yaml

```diff
@@ -19,8 +22,9 @@ jobs:
       id-token: write
     env:
       CP_HOME_DIR: "~"
+      BUNDLE_GEMFILE: '${{ env.cocoapods_dir }}/Gemfile'
```

Comment:
- [ ] you can set directly here the to `CocoaPods/Gemfile` and dismiss the global env.

Reply:
Done!